sct-commons/src/main/java/org/lfenergy/compas/sct/commons/scl/ldevice/LDeviceAdapter.java

```diff
@@ -169,8 +170,8 @@ public String getXPath() {
      */
     public void updateLDName() throws ScdException {
         String newLdName = parentAdapter.getCurrentElem().getName() + currentElem.getInst();
-        if (newLdName.length() > 33) {
-            throw new ScdException(newLdName + "(IED.name + LDevice.inst) has more than 33 characters");
+        if (newLdName.length() > MAXIMUM_LENGHT_LD_NAME_BY_XSD) {
```

Comment:
As the [maxLength](https://github.com/com-pas/compas-scl-xsd/blob/develop/src/main/resources/xsd/SCL2007B4/SCL_BaseSimpleTypes.xsd#L102C8-L102C17) known as XSD constraint I think we shouldn't even care about verifying this.

Reply:
Okay, In order to maintain traceability I would recommend discussing this with @hkaroun for related issues like https://github.com/com-pas/compas-sct/issues/7 and Jira tickets.

Reply:
After discussing together, I will completely remove the check